vector/src/main/java/im/vector/fragments/PeopleFragment.java

```diff
@@ -152,13 +155,15 @@ public void onResume() {
         super.onResume();
         mSession.getDataHandler().addListener(mEventsListener);
         ContactsManager.getInstance().addListener(this);
-        // Direct chats
-        initDirectChatsData();
-        initDirectChatsViews();
 
         // Local address book
         initContactsData();
+
+        // Direct chats
+        initDirectChatsData();
+
         initContactsViews();
+        initDirectChatsViews();//todo: to be removed
```

Comment:
If I understand correctly, this call is useless. We load the direct chats in the adapter, whereas we won't use them.
We should not reuse PeopleFragment/PeopleAdapter to handle the Tchap Contacts screen. 2 new files should be created. Do you agree with this?